CONTRIBUTING.md

```diff
@@ -74,3 +74,5 @@ We're following the [Google Go Code Review](https://code.google.com/p/go-wiki/wi
 + If you do not have write access to the repository and your pull request requires a manual merge, you may be asked to rebase again,
   `git fetch origin; git rebase -i origin/master` and update the PR again. Otherwise, you are free to merge your branch into origin/master directly or rebase first as you deem appropriate.
 
++ If you get a test failure in CircleCI, check the error log obtained from https://circle-artifacts.com/gh/cockroachdb/cockroach/$CIRCLE_BUILD_NUM/artifacts/0/$CIRCLE_ARTIFACTS/excerpt.txt. `$CIRCLE_BUILD_NUM` and `$CIRCLE_ARTIFACTS` can be found at the beginning of the test log. 
```

Comment:
The test failures tab (when it works, which is not always) is often better than excerpt.txt since it breaks things out by test method and includes the complete logs for those methods that fail.